### DISCUSSION TITLE
가로 세로 변경

### DISCUSSION THREAD

**Discussion by jun5640** [naverz/zepeto-studio-kor#109] [5. Q&A]
다른 월드에서 보면 아래쪽에 가로 세로 모드 변경이 가능하던대 해당 기능과 관련된 API 혹은 예제 코드를 알 수 있을까요?

**Comment by cheolhui-lee** [comment]
안녕하세요.
기존 월드에서 제공하던 가로/세로 전환 기능은 제페토 스튜디오에서는 현재 제공하고 있지 않습니다.
스튜디오로 만든 모든 월드는 고정 형태인 점 참고해주세요.
고맙습니다.
